workshopapps/engineerprofile360.web
# Set Employee Password

Task reference

This section contains a link to a corresponding task in Linear.
https://linear.app/team-axle-engineer-360-project/issue/BAC-40/setemployeepassword
Summary

This section contains a brief description of what was accomplished within the current PR.
Implementation details

This section describes the technical aspects of the solution, implemented withing the current PR, that are worth mentioning to the reviewers.
I implemented employee password set. An enrolled employee provides email to identify the employee and desired password to set the password
How to test

This section contains a checklist, describing basic scenarios that would help verify that the submitted code works correctly.
make a post request to /employee/update endpoint
supply parametres (email, password)
get response
References

Any additional information (screenshots, video, diagrams) that would help understand and review the PR.
